org-formation/650-identity-providers/_tasks.yaml

```diff
@@ -645,14 +645,14 @@ GithubOidcWorkflowsProdNextflowInfra:
       - !Ref WorkflowsNextflowProdAccount
     Region: us-east-1
 
-GithubOidcSageBionetworksMTBSageIT:
+GithubOidcBridgeDigitalHealthOpenBridgeWebSageIT:
   Type: update-stacks
   DependsOn: GithubOidcSageBionetworks
   Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.7.6/templates/IAM/github-oidc-provider.j2
-  StackName: !Sub ${resourcePrefix}-${appName}-sage-bionetworks-mtb-sageit
+  StackName: !Sub ${resourcePrefix}-${appName}-bridgedigitalhealth-openbridgeweb-sageit
   Parameters:
     ProviderArn: !CopyValue [ !Sub '${resourcePrefix}-${appName}-ProviderArn' ]
-    ProviderRoleName: !Sub ${resourcePrefix}-${appName}-sage-bionetworks-mtb-sageit
+    ProviderRoleName: !Sub ${resourcePrefix}-${appName}-bridgedigitalhealth-openbridgeweb-sageit
```

Comment:
i think this is getting close to the 64 char limit, i would drop `sageit` at the end here and on the `StackName:`

Reply:
Thanks, removed trailing sageit everywhere.